src/Analyzer.Core.JsonRuleTests/README.md

```diff
@@ -0,0 +1,46 @@
+# Test Project for Built-in JSON Rules
+This project is designed for quickly adding tests to verify the correctness of built-in JSON rules.  When new rules are added, tests should be added here to make sure the rule is written correctly and help protect against incorrect changes to the rule.
+
+## Test Setup
+A test (or tests) for a given JSON rule consists of 2 parts:
+1. A JSON configuration describing the test(s).
+2. One or more test ARM templates to analyze.  The results of the analysis are compared with the test configuration to assert correctness of the JSON rule.
+
+### JSON Configuration
+To create a set of tests for a JSON rule, a new JSON file is created in the *Tests* directory.  The name of the file must be the same as the `name` property of the JSON rule, with ".json" as the file extension.
+
+For example, a JSON rule like the following:
+``` json
+{
+    "name": "SuperSecuryCheck",
+    "description": "...",
+    "recommendation": "...",
+    "helpUri": "...",
+    "evaluation": {
+      ...
+    }
+}
+```
+... would have tests defined in a configuration file named "Tests/SuperSecuryCheck.json".
+
+The JSON test configuration has the following schema:
+``` json
+[
+    {
+        "Template": "Name of template file to analyze (without file extension).",
+        "ExpectPass": "true/false - Whether the template named in 'Template' is expected to pass for this rule.",
```

Comment:
You might want to add descriptions as comments instead of the string value. This would make it more clear that ExpectPass is a boolean type.

Reply:
I considered that.  But the JSON parsing will parse a bool from a string without issues too, so I figured it didn't really matter much.